src/main/resources/MapMacro/Code/Configuration.xml

```diff
@@ -31,17 +31,17 @@
   <parent>xwiki:MapMacro.Code.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1530103687000</date>
-  <contentUpdateDate>1530102784000</contentUpdateDate>
+  <date>1531407225000</date>
+  <contentUpdateDate>1531406961000</contentUpdateDate>
   <version>1.1</version>
   <title>Configuration</title>
   <comment/>
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{velocity}}
+  <content>{{velocity wiki="true"}}
```

Comment:
This is not needed, as the default value for the `wiki` parameter is `true`. See: https://extensions.xwiki.org/xwiki/bin/view/Extension/Script%20Macro.